content/posts/scientific-python/will_tirone/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Your Code Could Go To Space"
+date: 2023-07-19
+draft: false
+tags: ["Open-source", "Scientific-Python", "NASA", "JPL"]
```

Comment:
```suggestion
tags: ["Open-source", "Scientific-Python", "NASA", "JPL", "scipy", "internship"]
```

Reply:
Should I commit these? Or do you have to, since you already approved the PR?

Reply:
Yes, please go ahead and commit the changes. There should be a button. For some reason, I don't seem to have permission to commit to your branch. Let me know if you run into issues.